service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.policy.service.policy;
+
+import bio.terra.policy.common.model.PolicyInput;
+import bio.terra.policy.common.model.PolicyName;
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+
+/** When we process a policy that is not known to TPS, then we use this combiner. */
+public class PolicyUnknown implements PolicyBase {
```

Comment:
Is this only for forcing conflict for tests, or is this needed outside of conflicts? If the former, could this be renamed to PolicyForceConflict or something?

Reply:
This is needed outside of conflicts. TPS cannot assume it knows about all policies.
I'm happy to rename it to something else!